definitions/checks/repositories/check_upstream_repository.rb

```diff
@@ -4,7 +4,7 @@ class Checks::CheckUpstreamRepository < ForemanMaintain::Check
     description 'Check if any upstream repositories are enabled on system'
     tags :pre_upgrade
     preparation_steps do
-      Procedures::Packages::Install.new(:packages => %w[yum-utils])
+      [Checks::CheckEpelRepository.new, Procedures::Packages::Install.new(:packages => %w[yum-utils])]
```

Comment:
Few times I thought of combining EPEL check and upstream repo check. Because on downstream we dont want any of these repositories ? what do you think ?

Reply:
In my opinion, for now we can keep this as it is because for remediation procedure it needs `yum-utils` package to be installed.